move/its/sources/trusted_addresses.move

```diff
@@ -0,0 +1,38 @@
+module its::trusted_addresses {
+    use sui::bcs::BCS;
+
+    const EMalformedTrustedAddresses: u64 = 0;
+
+    public struct TrustedAddresses has copy, drop {
+        trusted_chains: vector<vector<u8>>,
+        trusted_addresses: vector<vector<u8>>,
+    }
+
+    public fun peel(bcs: &mut BCS): TrustedAddresses {
```

Comment:
Instead of adding a special `TrustedAddresses` struct, can we just submit multiple set_trusted_address calls? Doesn't need to block this PR though

Reply:
since these are each a gateway Message, it would be a bit of a pain for axelar to manage, and would make it more prone to errors.

Reply:
Also, for the abi comment, it is BCS that does not have peel_vec_string, not abi.